src/Logary/Internals/SpanModule.fs

```diff
@@ -90,38 +81,32 @@ module SpanBuilder =
       do t <- {t with clock = clock}
       x
 
-    member x.withLocalRootPrefix prefix =
-      if not <| String.IsNullOrEmpty(prefix) then
-        do t <- {t with localRootPrefix = prefix}
-      x
     member x.start () =
-      let spanId = generateId ()
+      let spanInfo = generateSpanInfo ()
       let beginAt = t.clock.GetCurrentInstant()
 
       let previous = activeSpan.Value
-      let bookMark = {
-        spanId = spanId
-        childCount = ref 0L
-      }
-      activeSpan.Value <- bookMark
+      activeSpan.Value <- spanInfo
```

Comment:
I don't like global statics at all... Globals go in the `Global` module, nowhere else, if possible.